x-pack/test/security_solution_cypress/cypress/e2e/ai4dsoc/navigation/navigation.cy.ts

```diff
@@ -147,9 +147,9 @@ describe('AI4dSoC Navigation', { tags: '@serverless' }, () => {
       cy.get(AI_SOC_NAVIGATION).should('exist');
 
       visibleLinks.forEach((link) => {
-        cy.getByTestSubjContains(`nav-item-id-${link}`).click();
```

Comment:
getByTestSubjContains matches both `configuration` and `confugration-integration`. It receives two elements here and failed the test.